terraform/module-ses/redis.tf

```diff
@@ -0,0 +1,55 @@
+resource "aws_route53_record" "redis" {
+  count   = "${var.create_elasticache == "true" ? 1 : 0}"
+  zone_id = "${var.zone_id}"
+  name    = "${var.project}-${var.elasticache_engine}"
+  type    = "CNAME"
+  ttl     = "300"
+  records = ["${aws_elasticache_cluster.redis.0.cache_nodes.0.address}"]
+}
+
+resource "aws_security_group" "redis" {
+  count       = "${var.create_elasticache == "true" ? 1 : 0}"
+  name        = "${var.project}-${var.elasticache_engine}-${var.env}"
+  description = "${var.elasticache_engine} ${var.env} for ${var.project}"
+  vpc_id      = "${var.vpc_id}"
+
+  ingress {
+    from_port = "${var.elasticache_port}"
+    to_port   = "${var.elasticache_port}"
+    protocol  = "tcp"
+
+    security_groups = "${var.elasticache_security_groups}"
+  }
+
+  tags {
+    Name       = "${var.project}-${var.elasticache_engine}-${var.env}"
+    client     = "${var.customer}"
+    env        = "${var.env}"
+    project    = "${var.project}"
+    role       = "${var.elasticache_engine}"
+    cycloid.io = "true"
+  }
+}
+
+resource "aws_elasticache_cluster" "redis" {
```

Comment:
ok for aws_elasticache_cluster. Maybe in the future use a replication (even if it is one, it could allow us to have password auth on redis)